conf/gulp-tasks/library.gulpfile.js

```diff
@@ -52,26 +92,19 @@ function compileCSS () {
     .pipe(dest('./dist/'));
 }
 
-function watchJS (cb) {
+/**
+ * Creates a new legacy bundle on JS file updates
+ *
+ * @param {BundleTaskFactory} bundleTaskFactory
+ */
+function watchJS (bundleTaskFactory) {
   return watch(['./src/**/*.js'], {
     ignored: './dist/'
-  }, parallel(legacyBundleIIFE));
```

Comment:
I think we can just remove the parallel here (I think I wrote this dingus line lol)

Reply:
lolll I'll remove!